src/templates/en/2019/methodology.html

```diff
@@ -124,7 +124,7 @@ <h2 class="header">Index</h2>
     <h1 class="title">Methodology</h1>
 
     <section>
-      <img class="content-banner" src="/static/images/methodology-banner.png" alt="">
+      <img loading="lazy" class="content-banner" src="/static/images/methodology-banner.png" alt="">
```

Comment:
Wouldn't bother lazy loading these:
```suggestion
      <img class="content-banner" src="/static/images/methodology-banner.png" alt="">
```